Common/src/main/java/com/blamejared/crafttweaker/impl/plugin/crafttweaker/BuiltinCraftTweakerPlugin.java

```diff
@@ -0,0 +1,87 @@
+package com.blamejared.crafttweaker.impl.plugin.crafttweaker;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.CraftTweakerConstants;
+import com.blamejared.crafttweaker.api.plugin.CraftTweakerPlugin;
+import com.blamejared.crafttweaker.api.plugin.IBracketParserRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.ICraftTweakerPlugin;
+import com.blamejared.crafttweaker.api.plugin.IJavaNativeIntegrationRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IListenerRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.ILoaderRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IRecipeHandlerRegistrationHandler;
+import com.blamejared.crafttweaker.api.plugin.IScriptLoadSourceRegistrationHandler;
+import com.blamejared.crafttweaker.api.tag.manager.ITagManager;
+import com.blamejared.crafttweaker.api.tag.manager.TagManagerWrapper;
+import com.blamejared.crafttweaker.api.tag.registry.CrTTagRegistryData;
+
+@CraftTweakerPlugin(CraftTweakerConstants.MOD_ID + ":builtin")
```

Comment:
We may need plugins for Forge and Fabric specific things.

We could use ServiceLoaders or we could have multiple plugins for each plugin?

If we have one plugin for each modloader, should we do:
```
crafttweaker:builtin
crafttweaker:builtin_forge
crafttweaker:builtin_fabric
```
or:
```
crafttweaker:builtin/common
crafttweaker:builtin/forge
crafttweaker:builtin/fabric
```

Reply:
I'd go for the multiple plug-ins approach. There isn't really a need to use another service loader, since we discover those automatically anyway. As for the naming, what about the following? ```
crafttweaker:builtin
crafttweaker:forge
crafttweaker:fabric
```